docs/docs/icicle/integrations.md

```diff
@@ -0,0 +1,97 @@
+# ICICLE integrated provers
+
+ICICLE has been used by companies and projects such as [Celer Network](https://github.com/celer-network), [Consensys Gnark](https://github.com/Consensys/gnark), [EZKL](https://blog.ezkl.xyz/post/acceleration/) and others to accelerate their ZK proving pipeline.
```

Comment:
I thought wanted to remove the projects names?

Reply:
No i said only companies i added projects